### PR TITLE
GitHub Actions: Fix class installation

### PR DESCRIPTION
**Pull request by hsinfan1996**
libomp was not found. Add one more LDFLAGS to fix the problem.

### CODE REVIEW

**Comment by hsinfan1996** [comment]
libomp was not found. Add one more LDFLAGS to fix the problem.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,17 +78,15 @@ jobs:
 
       - name: install class
         run: |
-          export LDFLAGS=-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib
           if [[ ${MATRIX_OS} == "macos-11" ]]; then
+            export LDFLAGS=-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib
+            LDFLAGS+=" -L/Users/runner/miniconda3/envs/test/lib"
```
Comment (damonge) [inline .github/workflows/ci.yml:83]: Don't you need to do
```
export LDFLAGS+=" -L/Users/runner/miniconda3/envs/test/lib"
```

Comment (hsinfan1996) [reply .github/workflows/ci.yml:83]: I also tried with zsh and I got error when doing `export LDFLAGS+=" -L/Users/runner/miniconda3/envs/test/lib"`

Comment (hsinfan1996) [reply .github/workflows/ci.yml:83]: interestingly, it is okay with bash.

Comment (hsinfan1996) [reply .github/workflows/ci.yml]: I am using something that works for both shells.

Comment (damonge) [reply .github/workflows/ci.yml]: Interesting! OK, as long as it works I'm happy.

**Comment by hsinfan1996** [comment]
Look like it is working.

**Review by damonge — APPROVED** [review]
Thanks a lot. LGTM!

**Pull request merged**